.github/workflows/deploy.yml

```
@@ -63,10 +63,11 @@ jobs:
             echo "${{ secrets.ENV_VARS }}" > ./.env
             
             
-            echo "🧹 Cleaning up existing containers and volumes..."
+            echo "🧹 기존 컨테이너 중지 및 삭제 중..."
             docker compose down -v 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)